README.md

```diff
@@ -1,2 +1,45 @@
-# my-Portfolio
- my portfolio website
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#   My Portfolio website
+
+- Project created to display my porfolio and to show my works
+
+
+## Built With
+
+- HTML & CSS
+- Linters
+
+
+## Authors
```

Comment:
- [ ] As stated by the previous reviewer he said replace so kindly delete line 14.

Reply:
@Kossi-stack, line 14 is the heading for the `Authors` section, it is not supposed to be deleted. The previous reviewer only suggested that @maccony202 should replace the generic name `Author1` with a proper name. @maccony202, I am sorry this has made your README look not organized. Please consider reverting this change.